libcontainer/container_linux.go

```diff
@@ -2195,6 +2195,9 @@ func (c *linuxContainer) bootstrapData(cloneFlags uintptr, nsMaps map[configs.Na
 		var mounts []byte
 		for _, m := range c.config.Mounts {
 			if m.IsBind() {
+				if strings.Contains(m.Source, "\x00") {
```

Comment:
In other places (e.g. https://github.com/opencontainers/runc/blob/0d5ac13200f0b12198f9a84d3c64942c3cd1637d/libcontainer/init_linux.go#L126-L131) we use `strings.IndexByte(str, 0) >= 0` which is slightly more straightforward/explicit.

Maybe it's better to use it here, too.

Reply:
:+1: